worldedit-core/src/main/java/com/sk89q/worldedit/command/UtilityCommands.java

```diff
@@ -1034,7 +1034,30 @@ public void accept(File file) {
         if (playerFolder) {
             if (listMine) {
                 File playerDir = MainUtil.resolveRelative(new File(dir, actor.getUniqueId() + dirFilter));
+                //FAWE start - Schematic list other permission
+                if (!actor.hasPermission("worldedit.schematic.list.other") && java.util.regex.Pattern
+                        .compile("[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}") // See SchematicCommands: 361
```

Comment:
It would be more efficient to have the pattern statically pre-compiled somewhere. Perhaps in the StringMan class for StringMan#isUUID(String)